### DISCUSSION TITLE
Run multiple instances of same application on demand

### DISCUSSION THREAD

**Discussion by antonholovko-cloud** [microsoft/aspire#5562] [Q&A]
Hi team, 

Are there any chances that you'll implement the ability to run the app on demand or have this in your roadmap.

For example: I need to run console app 5 times but with different parameters within aspire infrastructure.

**Comment by davidfowl** [comment]
You can write a loop and add 5 project resources with different names (but using the same project).

**Comment by antonholovko-cloud** [comment]
Hi David, thank you for response! 

Is there any way to launch a resource on demand?

**Reply by Alirexaa** [reply]
related #295

**Comment by antonholovko-cloud** [comment]
I was thinking about running 5 instances of the app that will listen to event/input, let me know if this is the path I need to follow.
